### DISCUSSION TITLE
Specify config file location / config directory?

### DISCUSSION THREAD

**Discussion by parasiteoflife** [rustic-rs/rustic#1155] [Q&A] [answered]
Is there a way to do this? I want to use a custom path for the config file but I couldn't find any command in the help, is this possible?

Edit: So I just found out that we can use a path without the extension i.e. `A:\configFolder\configFile` and it will load it, is this intended?
Also when I used the full path (with extension) rustic told me that it couldn't find the config file in 3 locations, they all three were the same paths `A:\configFolder\configFile.toml.toml`

**Comment by aawsome** [comment, marked as answer]
>So I just found out that we can use a path without the extension i.e. A:\configFolder\configFile and it will load it, is this intended?

Yes, it is intended that if you don't use one of the three config folders, you may give a config profile file directly (was mainly intended to use some in the local dir, but also works with all relative and absolute paths)

**Reply by s-leroux** [reply]
As a follow-up, how are relative paths resolved in the `use-profiles` option?

It seems to me they end up being resolved relative to the process working directory (like with multiple `-P` options).
1. Is there a way to specify a profile path relative to the location of the including profile?
2. Is there a way to specify the profile search paths (like the PATH environment variable)?
